packages/documentation-site/patternfly-docs/content/AI/conversation-design/chatbots.md

```diff
@@ -0,0 +1,64 @@
+---
+id: Chatbots
+section: AI
+subsection: Conversation design
+---
+
+import { Button, Card, CardBody, CardFooter, CardTitle, Divider, Gallery, GalleryItem } from '@patternfly/react-core';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+
+## Writing for chatbots 
+
+When chatbots are design to meet the needs of your users, they can improve the overall UX of your product, because they are convenient, efficient, and persistent. 
```

Comment:
```suggestion
When chatbots are designed to meet the needs of your users, they can improve the overall UX of your product, because they are convenient, efficient, and persistent. 
```